src/main/java/de/tum/in/www1/artemis/web/rest/ExamResource.java

```diff
@@ -356,6 +363,22 @@ private void checkExamPointsAndCorrectionRoundsElseThrow(Exam exam) {
         }
     }
 
+    /**
+     * GET /exams/all : Find all exams the user is allowed to access
+     *
+     * @param pageable pageable parameters
+     * @return the ResponseEntity with status 200 (OK) and a list of exams. The list can be empty
+     */
+    @GetMapping("exams/all")
```

Comment:
Where is the difference to the method below (endpoint `GET /exams`)?

Reply:
I would suggest, we limit this to the currently active ones, otherwise, too many not interesting exams would be returned.
Currently could e.g. mean: find exams where the visible date is between "now - 7days" and "now + 7 days"

Additionally, we should limit this to instructors

Reply:
@ge65cer The method below is used to search exams (e.g. by title) and has mandatory fields that need to be used. And i could not reuse it.

Reply:
> I would suggest, we limit this to the currently active ones, otherwise, too many not interesting exams would be returned. Currently could e.g. mean: find exams where the visible date is between "now - 7days" and "now + 7 days"
> 
> Additionally, we should limit this to instructors

@krusche  I will add two new RequestParam (fromDate and toDate) which will contain dates and i can query the exams between dates. 
And this `"now - 7days" and "now + 7 days"` logic can be implemented in IPad part.